test/e2e/basic/specialresource_test.go

```diff
@@ -1,19 +1,23 @@
 package e2e
 
 import (
+	"os"
 	"testing"
 
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
 	"github.com/openshift-psap/special-resource-operator/test/framework"
 )
 
+var ns string
```

Comment:
How about
```suggestion
var ns = os.Getenv("NAMESPACE")
```
Also, is there any way we could pass the namespace without using a global variable?

Reply:
It could work.

Regarding not using global variable we will need then to `os.Getenv()` everywhere in the test files of the same pkg instead of using the variable.

IMO, we don't want to have multiple places in code that refers to env variables, ideally you initialise it in a config struct in the code and populate it to the consumers.

Reply:
I agree with not having `os.Getenv` everywhere. But could we pass `ns` as a variable to all the functions that need it?

Reply:
Yes, I have created a `Config` struct for `clientSet` and added the namespace to that config object.